src/weapon.py

```diff
@@ -117,9 +115,11 @@ def update(
         self.time_since_shoot += delta_time
 
     def shoot(self, projectile_list: arcade.SpriteList):
-        rocket = arcade.SpriteSolidColor(30, 20, arcade.color.ORANGE)
+        rocket: SpriteForRocket = SpriteForRocket(self)
         offset_sprite_from(rocket, self.weapon_sprite, self.muzzle_offset)
         rocket.velocity = polar_to_vec(self.rocket_speed, self.weapon_sprite.radians)
+        #Corrects for rocket texture angled up 45 degrees 
+        rocket.angle -= 45
```

Comment:
Nit: move this to `SpriteForRocket` constructor?  Doesn't need to change now because I suspect we'll be cleaning up this sprite stuff anyway when Liam's collision code gets in.

Reply:
Fixed. I made it the angle for `muzzle_offset`

Reply:
Oh right!  Smart